internal/cli/deployments/setup.go

```diff
@@ -16,21 +16,17 @@ package deployments
 
 import (
 	"context"
-	_ "embed"
```

Comment:
[q]I remember at some point we removed the local dev tests/marked them to spot running as they're flaky. I strongly recommend we add them back sooner rather than later so that we can see easily how our changes may affect the functionality, do we have a ticket for that?

Reply:
Do you know which tests so I can create a ticket?